test/alternator/test_ttl.py

```diff
@@ -189,17 +190,28 @@ def test_update_ttl_errors(dynamodb):
 
 # Basic test that expiration indeed expires items that should be expired,
 # and doesn't expire items which shouldn't be expired.
-# This is an extremely slow test - DynamoDB documentation says that
+# On AWS, this is an extremely slow test - DynamoDB documentation says that
 # expiration may even be delayed for 48 hours. But in practice, at the time
 # of this writing, for tiny tables we see delays of around 10 minutes.
 # The following test is set to always run for "duration" seconds, currently
-# 15 minutes. During this time, we expect to see the items which should
+# 20 minutes on AWS. During this time, we expect to see the items which should
 # have expired to be expired - and the items that should not have expired
 # should still exist.
+# When running against Scylla configured (for testing purposes) to expire
+# items with very short delays, "duration" can be set much lower so this
+# test will finish in a much more reasonable time.
 @pytest.mark.veryslow
 @pytest.mark.xfail(reason="TTL not implemented yet #5060")
 def test_ttl_expiration(dynamodb):
-    duration = 900
+    if is_aws(dynamodb):
+        duration = 1200
+    else:
+        duration = 3
```

Comment:
note: Python has a syntax that I personally find very nice and concise:
```python
duration = 1200 if is_aws(dynamodb) else 3
```
(Feel free to ignore this comment if you prefer the expanded version, I don't deeply mind either way)

Reply:
You're right. I'll use that syntax.